src/datasources/flow-ds/datasource.ts

```diff
@@ -441,10 +441,11 @@ export class FlowDatasource {
     }
 
     if (combineIngressEgress) {
+      const ingressColumns =  flowSeries.columns.filter(column => column.ingress);
+      const egressColumns =  flowSeries.columns.filter(column => !column.ingress);
+      const columns = ingressColumns.length >= egressColumns.length ? ingressColumns : egressColumns;
```

Comment:
Better to use a unique combination of both?

Reply:
What I noticed was that ingress/egress flows are not always even in quantity. And when using the combineIngressEgress it was inmediatelly filtering the result by using ingress as the # of columns. 

If the # of columns has only egress flows or more egress than ingress then the result  will only consider ingress to iterate.

I was pulling some data from the demo site and I notice the flows for example returned are only ingress. 
So if it was the opossite scenario (only egress flows) then it would not be any data to display. This way I'm using the largest number of columns to decide how iterate over all the possible values and do the calculation.

Reply:
> Better to use a unique combination of both?

What I noticed was that ingress/egress flows are not always even in quantity. And when using the combineIngressEgress it was inmediatelly filtering the result by using ingress as the # of columns.

If the # of columns has only egress flows or more egress than ingress then the result will only consider ingress to iterate.

I was pulling some data from the demo site and I notice the flows for example returned are only ingress.
So if it was the opossite scenario (only egress flows) then it would not be any data to display. This way I'm using the largest number of columns to decide how iterate over all the possible values and do the calculation.

Reply:
I tacked this separately since Will was having same results. Insetad of filtering by columns by ingres/egress I just created a Set of unique columns from the series, avoiding any possible mismatching scenario.